Readme.rst

```diff
@@ -20,3 +20,19 @@ Api
 There exists an api to get all the events as json format. You can call this api with a special type. Just attach
 "?type=1452982642" to your url and typoscript will handle the rest like calling the extension "slubevents" and the
 plugin "apieventlist".
+
+You can influence the result of events with the following additional parameter:
+
+* tx_slubevents_apieventlist[category]: comma separated string, list of category ids
+* tx_slubevents_apieventlist[discipline]: comma separated string, list of discipline ids
+* tx_slubevents_apieventlist[contact]: comma separated string, list of contact ids
+* tx_slubevents_apieventlist[showPastEvents]: 0|1, in default (0) events are shown with beginning today
+* tx_slubevents_apieventlist[showEventsFromNow]: 0|1, additional setting for "showPastEvents"
+* tx_slubevents_apieventlist[limitByNextWeeks]: integer, set a limit for the next weeks
+* tx_slubevents_apieventlist[startTimestamp]: timestamp, influence the start date, works together with stopTimestamp
+* tx_slubevents_apieventlist[stopTimestamp]: timestamp, influence the stop date, works together with startTimestamp
+* tx_slubevents_apieventlist[sorting]: asc|desc, default value is ascending
+* tx_slubevents_apieventlist[limit]: integer, limitation of result data
+
+If you use these parameter add "tx_slubevents_apieventlist" in [FE][cacheHash][cachedParametersWhiteList] and
+[FE][cacheHash][excludedParameters].
```

Comment:
This documentation should go to the extension documentation. This can be done later, when this feature #76 is done.